src/intrprtr.c

```diff
@@ -4009,11 +3891,9 @@ static StructInitInfo module = {
     .type = MODULE_BUILTIN,
     .name = "intrprtr",
     .initKernel = InitKernel,
-
-    .initModuleState = InitModuleState,
```

Comment:
Since `InitModuleState` is gone, we can and must remove this.

I believe I've explained all changes in `intrprtr.c` now in one way or another.